src/component/tooltip/TooltipHTMLContent.ts

```diff
@@ -60,28 +60,31 @@ function mirrorPos(pos: string): string {
 }
 
 function assembleArrow(
-    backgroundColor: ColorString,
+    tooltipModel: Model<TooltipOption>,
     borderColor: ZRColor,
     arrowPosition: TooltipOption['position']
 ) {
+    const backgroundColor = tooltipModel.get('backgroundColor');
+    const borderWidth = tooltipModel.get('borderWidth');
+
     if (!isString(arrowPosition) || arrowPosition === 'inside') {
         return '';
     }
 
     borderColor = convertToColorString(borderColor);
     const arrowPos = mirrorPos(arrowPosition);
-    let positionStyle = `${arrowPos}:-6px;`;
+    let positionStyle = `${arrowPos}:-10px;`;
```

Comment:
What is this change for?

Reply:
When I increased the `borderWidth` of the arrow, the placement is slightly off. So in order to adjust it I made this change.
If there is any better way please suggest me.

Thanks....

Reply:
If it is related to user-set `borderWidth`. I think it's better to calculate this value from `broderWidth`

Reply:
Yeah, on it.
Since the square of the arrow is rotated need to calculate the diagonal.

Reply:
Fixed it :exclamation: